### PR TITLE
Polish Javadoc and inline comments

### PR DESCRIPTION
**Pull request by yuhangbin**
Fix some typos and polish trivial stuff.

### CODE REVIEW

File: spring-test/src/main/java/org/springframework/test/context/ContextConfiguration.java [hunk 1]
```diff
@@ -89,7 +89,7 @@
  * @see SmartContextLoader
  * @see ContextConfigurationAttributes
  * @see MergedContextConfiguration
- * @see org.springframework.context.ApplicationContext ApplicationContext
+ * @see org.springframework.context.ApplicationContext
```
Comment (sbrannen) [inline spring-test/src/main/java/org/springframework/test/context/ContextConfiguration.java:92]: This is intentional. Please revert that change.

File: spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java [hunk 2]
```diff
@@ -755,7 +755,7 @@ protected void invokeBeanFactoryPostProcessors(ConfigurableListableBeanFactory b
 		PostProcessorRegistrationDelegate.invokeBeanFactoryPostProcessors(beanFactory, getBeanFactoryPostProcessors());
 
 		// Detect a LoadTimeWeaver and prepare for weaving, if found in the meantime
-		// (e.g. through an @Bean method registered by ConfigurationClassPostProcessor)
+		// (e.g. through a @Bean method registered by ConfigurationClassPostProcessor)
```
Comment (sbrannen) [inline spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java:758]: When read as "at Bean", the use of "an" is correct.

Thus, please revert this change.

File: spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java [hunk 3]
```diff
@@ -112,7 +112,7 @@
  *
  * <p>Implements resource loading by extending
  * {@link org.springframework.core.io.DefaultResourceLoader}.
- * Consequently treats non-URL resource paths as class path resources
+ * Consequently, treats non-URL resource paths as class path resources
```
Comment (sbrannen) [inline spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java:115]: The prose here uses verbal phrases. Thus "consequently" is an adverb for "treats" and does not need a comma.

So please revert this change.

File: spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java [hunk 4]
```diff
@@ -284,7 +284,7 @@ public String getApplicationName() {
 
 	/**
 	 * Set a friendly name for this context.
-	 * Typically done during initialization of concrete context implementations.
+	 * Typically, done during initialization of concrete context implementations.
```
Comment (sbrannen) [inline spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java:287]: Please revert this change.

**Review by sbrannen — CHANGES_REQUESTED** [review]

File: spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java (hunk 4 above)
Comment (yuhangbin) [reply spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java:287]: I will revert.

File: spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java (hunk 3 above)
Comment (yuhangbin) [reply spring-context/src/main/java/org/springframework/context/support/AbstractApplicationContext.java:115]: Thank you for telling me. I will revert

**Comment by yuhangbin** [comment]
Already revert.

**Review by sbrannen — APPROVED** [review]

**Comment by sbrannen** [comment]
Thanks for making the requested changes.

This has been merged into `main`.
